docker/shadow-hand/indigo/nvidia_ide/Dockerfile

```diff
@@ -0,0 +1,16 @@
+#
+# Indigo with build tools Dockerfile
+#
+# https://github.com/shadow-robot/sr-build-tools/
+#
+
+FROM shadowrobot/dexterous-hand:indigo
+
+LABEL Description="This image is used to make ROS Indigo based projects build faster using build tools. It includes IDE environments. Nvidia compatible" Vendor="Shadow Robot" Version="1.0"
```

Comment:
Description doesn't reflect purpose of this image. It is copied from top level image :)